### ISSUE TITLE
Default auto-battle (x1.5?) speedup is disabled

### ISSUE THREAD

**Issue by Ryunam** [Albeoris/Memoria.FFPR#11]
Thank you first of all for developing this awesome tool! It's an amazing resource and it's making the prospect of editing the text assets a thousand times easier.

One little thing I have noticed while using this plugin is that the normal in-battle speedup, which would occur whenever using the default auto-battle function, does not seem to work anymore. The speedup toggle offered with Memoria works well and the auto-battle system itself is triggered as normal whenever you press the Square (PS) / X (Xbox) button while in battle, however the default x1.5 battle speedup does not happen at all.

Is this intentional? It would be great if possible to still retain the default functionality of auto-battle + speedup as provided by default in the game, so that it's still possible to use the default mechanisms if one prefers to do so, while still having Memoria installed in the game directory. Hope this is feasible!

**Comment by Albeoris** [comment]
> however the default x1.5 battle speedup does not happen at all.

Hi! Can you explain about this feature little more? I didn't know about it, so I can't say how difficult it is to support this functionality.

**Comment by Ryunam** [comment]
Of course! To better illustrate the issue that I am seeing, I have recorded two videos that show the difference between:
1) the default game with no plugin installed and 
2) the game with Memoria installed.

-----

**1) This is with the default behavior of the game with Memoria not installed.**
When I press Square on my PS controller (or the corresponding key on the keyboard) while in a battle, Auto-battle is activated and the overall speed of the battle is increased. Pay attention to how the sword slashes and attack hits go quickly one after another.

https://user-images.githubusercontent.com/17614150/128494192-50ccbf7c-1e67-47c2-8381-c1501a2d9256.mp4

**2) This is with Memoria installed in the game directory**
When I press Square on my PS controller (or the corresponding key on the keyboard) while in a battle, Auto-battle is activated but only with regards to the command input. The overall speed of the battle is left unchanged. Notice how, compared to the previous video, the pacing of the attacks is considerably slower.

https://user-images.githubusercontent.com/17614150/128495173-915ff729-5123-4513-a0fd-435ec90b1cd1.mp4

In the first video I take about 9-10 seconds to kill 5 goblins (from 0:02 to 0:11), while in the second video I take about 10 seconds to kill only **4** goblins (from 0:02 to 0:12).

Hope this helps to clarify the problem!

**Comment by granfalloon** [comment]
Just wanted to chime in that I'm experiencing the same issue, game doesn't speed up with auto-battle anymore since installing Memoria to play the NES Rebalance mod.

**Comment by devonfae** [comment]
Here's a quick overview of how the Auto-Battle toggle works in the game, including the addresses of the relevant calls:
https://steamcommunity.com/sharedfiles/filedetails/?id=2565514151
Notably, the speedup code path and the auto-input code path are separate, so whatever Memoria is doing, it's triggering the one without the other.

**Comment by Eatitup86** [comment]
I wanted to note that the 1X speed actually seems slower than without Memoria as well which kind of forces the player to leave speed up on when using it.

**Comment by Ryunam** [comment]
I did not notice any apparent speed difference  between the default game and a Memoria installation while fighting normally without auto-battle. Perhaps a recording would be helpful to have a direct comparison.

What I guess is happening here anyway is that the speedup toggle added with Memoria might be overriding the normal speed switching that occurs in the game when using auto-battle.
